code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -110,6 +115,11 @@
 	if(!ishuman(owner))
 		return
 	var/mob/living/carbon/human/H = owner
+	var/datum/species/ipc/species_datum = H.dna.species
```

Comment:
Honestly I'd just add the same check to even giving the IPC the action in the first place here:

https://github.com/retlaw34/Shiptest/blob/9c84ad7525ebe5fed6e44f85103a80119b23663c/code/modules/mob/living/carbon/human/species_types/IPC.dm#L70-L72

Reply:
should be addressed now, the screen action button is never given now